ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -204,29 +377,35 @@ def get_mac(self, iff):
 
     def generate_from_t1(self):
         self.from_t1 = []
-        self.ip_addr = []
 
         vlan_ip_range = self.test_params['vlan_ip_range']
 
         _, mask = vlan_ip_range.split('/')
         n_hosts = min(2**(32 - int(mask)) - 3, self.max_nr_vl_pkts)
```

Comment:
>n_hosts = min(2**(32 - int(mask)) - 3, self.max_nr_vl_pkts) [](start = 8, length = 59)

i think it is better to simply the test, and we just define the number of hosts under the vlan and send 1 packet per host every iteration. 

sending multiple packet per host does not seem neccessary and it complicates the data interpretation later.